src/app/pages/index/components/list-requests/list-requests.component.ts

```diff
@@ -1,16 +1,65 @@
 import { Component, OnInit } from '@angular/core';
+import { DebugService } from '../../../../services/debug.service';
+import { MatIconRegistry } from '@angular/material/icon';
+import { DomSanitizer } from '@angular/platform-browser';
+import { Debug } from '../../../../models/Debug';
 
 @Component({
   selector: 'app-list-requests',
   templateUrl: './list-requests.component.html',
   styleUrls: ['./list-requests.component.css'],
 })
 export class ListRequestsComponent implements OnInit {
+  loading: boolean = false;
+  debugsList: Debug[] = [];
+  displayedColumns: string[] = ['position', 'tag', 'ip', 'method', 'isAjax', 'url', 'code', 'memory', 'time'];
 
-  constructor() {
+  constructor(private debugService: DebugService,
+              iconRegistry: MatIconRegistry,
+              sanitizer: DomSanitizer) {
+    iconRegistry.addSvgIcon(
+      'check',
+      sanitizer.bypassSecurityTrustResourceUrl('assets/img/yes-icon.svg'));
+    iconRegistry.addSvgIcon(
+      'ban',
+      sanitizer.bypassSecurityTrustResourceUrl('assets/img/no-icon.svg'));
   }
 
   ngOnInit(): void {
+    this.getDebugsList();
   }
 
+  getDebugsList() {
+    this.loading = true;
+    this.debugService.getList().subscribe(
+      response => {
+        this.loading = false;
+        this.debugsList = response;
+      }
+    );
+  }
+
+  formatMemory(memory: number): string {
+    return String((Number(memory / 1048576)).toFixed(0)) + ' MB';
```

Comment:
```suggestion
    return (Number(memory / 1048576)).toFixed(2) + ' MB';
```
There is no need to wrap to string object since toFixed returns a string.

Reply:
Ok. Will keep than in mind! Thanks for the advice!